src/Visualization/Python/Render1D.py

```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+# Python script to render 1D data written using SpECTRE in format v2
+
+import glob
+import h5py
+import argparse
+import sys
+import os
+import numpy as np
+import logging
+import matplotlib as mpl
+try:
+    import Tkinter as tkinter
+except ImportError:
+    import tkinter
```

Comment:
Why is the `try/except` necessary? Please add a comment

Reply:
The module name depends on the python version. I have added a comment above.